op-guide/configuration.md

```diff
@@ -173,7 +178,7 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 
 + the store capacity
 + default: 0 (unlimited)
-+ PD uses this flag to determine how to balance this TiKV.
++ PD uses this flag to determine how to balance this TiKV. (Tips: you can use "10GB" instead of 1073741824)
```

Comment:
What does "this TiKV refer to? 
(Tips: you can use "10GB" instead of 1073741824) -> Tip: you can use 10GB instead of 1073741824.